devenv/lib/colima.py

```diff
@@ -41,17 +39,29 @@ def uninstall(binroot: str) -> None:
             pass
 
 
-def install(
-    version: str, url: str, sha256: str, reporoot: Optional[str] = ""
-) -> None:
-    if reporoot:
-        binroot = fs.ensure_binroot(reporoot)
-    else:
-        # compatibility with devenv <= 1.4.0
-        binroot = f"{root}/bin"
-        os.makedirs(binroot, exist_ok=True)
+def install_shim(binroot: str) -> None:
+    fs.write_script(
+        f"{binroot}/colima",
+        """#!/bin/sh
+export COLIMA_HOME="{home}/.colima"
+
+# needed to ensure COLIMA_HOME is what we want it to be
+unset XDG_CONFIG_HOME
+
+exec "{binroot}/colima-bin" "$@"
+""",
+        shell_escape={"binroot": binroot, "home": home},
```

Comment:
this is going to double shell escape I think

Reply:
same for `home`

Reply:
```sh
#!/bin/sh
export COLIMA_HOME="/Users/josh/.colima"

# needed to ensure COLIMA_HOME is what we want it to be
unset XDG_CONFIG_HOME

exec "/Users/josh/dev/sentry/.devenv/bin/colima-bin" "$@"
```

Reply:
have a space in `home` or `binroot`

Reply:
hmm

```
fs.write_script(
        "test.sh",
        """#!/bin/sh
export fp="{dir}/foo"
touch "$fp"
""",
        shell_escape={"dir": "/Users/josh/s p a c e"},
    )
```

```
bash test.sh
touch: '/Users/josh/s p a c e'/foo: No such file or directory
```

would the expectation be

`export fp='/Users/josh/s p a c e/foo'`?

Reply:
ah

```
fs.write_script(
        "test.sh",
        """export fp="{dir}/foo"
mkdir -p "{dir}"
touch "$fp"
test -e "$fp" || echo fail
""",
        shell_escape={"dir": "/Users/josh/s p a c e"},
    )
```

this generates

```
export fp="'/Users/josh/s p a c e'/foo"
mkdir -p "'/Users/josh/s p a c e'"
touch "$fp"
test -e "$fp" || echo fail
```

which doesn't fail but it makes this dir haha...

```
josh@CTKP7QRX4L.local ~  $ tree \'                                                                                                                        0
'
└── Users
    └── josh
        └── s p a c e'
            └── foo
```

Reply:
need to unquote it seems

```
fs.write_script(
        "test.sh",
        """export fp={dir}/foo
mkdir -p {dir}
touch "$fp"
test -e "$fp" || echo fail
""",
        shell_escape={"dir": "/Users/josh/s p a c e"},
    )
```

...

```
josh@CTKP7QRX4L.local ~  $ tree s\ p\ a\ c\ e/                                                                                                            
s p a c e/
└── foo
```

Reply:
will apply this to other shims in a followup pr